### DISCUSSION TITLE
Laravel Forge and Git Hub

### DISCUSSION THREAD

**Discussion by cheintzelman141** [community/community#58866] [New to GitHub]
I use laravel forge to maintain my servers, I recently changed the source code owner to a new account under my company. When the owner changed I lost the keys to be able to push code from github to the servers. The support at forge told me to add the keys to git hub. When I attempt to do this, some keys will add, others tell me that they are in use. I have checked the members of my organization and they are not in use. Where can I check to see if they are in use and how can I add these keys back so I can get my team back up and running ?

**Comment by queenofcorgis** [comment]
Apologies for the delay, the best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Best of luck and thank you for your patience!
